library/core/src/slice/iter.rs

```diff
@@ -2967,3 +2968,167 @@ unsafe impl<'a, T> TrustedRandomAccess for IterMut<'a, T> {
         false
     }
 }
+
+/// An iterator over slice in (non-overlapping) chunks separated by a predicate.
+///
+/// This struct is created by the [`group_by`] method on [slices].
+///
+/// [`group_by`]: ../../std/primitive.slice.html#method.group_by
+/// [slices]: ../../std/primitive.slice.html
+#[unstable(feature = "slice_group_by", issue = "none")]
+#[derive(Debug)] // FIXME implement Debug to be more user friendly
```

Comment:
Will this get fixed after pull request or as part of the pull request?

Reply:
I marked it there, I will probably fix it in this PR, I forgot about that, thanks for reminding me.